### PR TITLE
chore: new runner 

### PR DESCRIPTION
**Pull request by NikolaMilosa**
Updating runner version and zig in that version

### CODE REVIEW

File: docker/runner.Dockerfile [hunk 1]
```diff
@@ -12,7 +12,7 @@ RUN apt-get update && \
     libssl-dev zlib1g-dev libbz2-dev libreadline-dev libsqlite3-dev libffi-dev liblzma-dev libncurses5-dev libncursesw5-dev \
     xz-utils tk-dev libffi-dev liblzma-dev python-openssl protobuf-compiler libdbus-1-dev softhsm2 libsofthsm2 opensc -y
 
-RUN curl -L https://ziglang.org/builds/zig-linux-x86_64-0.14.0-dev.321+888708ec8.tar.xz | tar -xJ && \
+RUN curl -L https://ziglang.org/builds/zig-linux-x86_64-0.14.0-dev.1713+f99097e68.tar.xz | tar -xJ && \
```
Comment (sasa-tomic) [inline docker/runner.Dockerfile:15]: Can we use the (more) stable 0.13 instead?
`https://ziglang.org/download/0.13.0/zig-linux-x86_64-0.13.0.tar.xz`

Comment (sasa-tomic) [inline docker/runner.Dockerfile:15]: Can we use the (more) stable 0.13 instead?
`https://ziglang.org/download/0.13.0/zig-linux-x86_64-0.13.0.tar.xz`

Comment (NikolaMilosa) [reply docker/runner.Dockerfile:15]: I've put 0.14

Comment (sasa-tomic) [reply docker/runner.Dockerfile:15]: I know but that's a dev build from some random revision, which obviously gets GCed after some time. Can we use 0.13 instead that is still available upstream, so is more likely not to disappear that quickly?

Comment (NikolaMilosa) [reply docker/runner.Dockerfile:15]: Oh its not stable. Sorry.... Changing... 😟

**Review by sasa-tomic — APPROVED** [review]

**Pull request merged**